stable/opencart/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: opencart
-version: 4.5.3
```

Comment:
This was not the previous version, I forgot to update my fork so I needed to update the version until the latest one.